frontend/src/index.jade

```diff
@@ -286,7 +286,9 @@ html
                         i.fas.fa-ellipsis-h.icon
                         span.tooltiptext.tooltiptext--close Show untouched code
                         span.tooltiptext.tooltiptext--open Hide untouched code
-                      .code {{ segment.lines.join("\n") }}
+                      .code(v-for="(line, index) in segment.lines")
+                        .line-number {{ segment.lineNumbers[index] + "\n" }}
+                        .line-content {{ line + "\n" }}
```

Comment:
hmm... this `lineNumbers` array shouldn't be necessary if jade can perform this functionality of pug.
https://pugjs.org/language/iteration.html#while

Can you please try it out?

Reply:
yep am on it.

Reply:
> hmm... this `lineNumbers` array shouldn't be necessary if jade can perform this functionality of pug.
> https://pugjs.org/language/iteration.html#while
> 
> Can you please try it out?

I think the `lineNumbers` array is necessary as there are un-authored and authored segments. We wouldn't know at which line the authored segments start without the array since there are un-authored segments which are not displayed unless you toggle them.